Dockerfile.test

```diff
@@ -0,0 +1,9 @@
+FROM python:3.13
```

Comment:
This is version 3.13 whereas you've set the `pyproject.toml` to 3.12. Intentional?